lib/web/sessions.go

```diff
@@ -79,6 +79,14 @@ func (c *SessionContext) String() string {
 	)
 }
 
+// GetAuthClient returns auth client
+func (c *SessionContext) GetAuthClient() (*auth.Client, error) {
```

Comment:
What's the difference between this and `GetClient` in the same file?

Reply:
@russjones removed